Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8.3 as builder
+WORKDIR /go/src/tiedot/
```

Comment:
You probably want to check the project out to it's fully qualified path `/go/src/github.com/HouzuoGuo/tiedot`, otherwise things may have trouble importing subpackages.